src/main/java/core/basesyntax/figures/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.figures;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length - 1)].toString();
```

Comment:
It would be nice not to make such complicated lines.
Suggestion:

int index = random.nextInt(Color.values().length);
return Color.values()[index].name();